File: 0001 Code Chef/0003 Good Turn/implementation/0005 kotlin/GoodTurn.kt
fun main(args:Array<String>){
    var stringInput:String? = readLine();
    if(stringInput != null){
        var stringArray:Array<String> = stringInput.split(" ").toTypedArray();
        var x:Int = stringArray[0].toInt();
        var y:Int = stringArray[1].toInt();
        var sumof:Int = x + y
        if(sumof > 6){
            println("YES");
        }else{
            println("NO");
        }
    }

}

File: 0001 Code Chef/0003 Good Turn/implementation/0003 c#/GoodTurn.cs
using System;

class GoodTurn{
    static void Main(string[] args){
        string stringInput = Console.ReadLine();
        string[] stringArray = stringInput.Split(' ');
        int x = Convert.ToInt32(stringArray[0]);
        int y = Convert.ToInt32(stringArray[1]);
        int sumof = x + y;
        if(sumof > 6){
            Console.WriteLine("YES");
        }else{
            Console.WriteLine("NO");
        }
    }
}